src/app/app.component.scss

```diff
@@ -0,0 +1,6 @@
+h1 {
+  margin-left: 40px;
+  margin-top: 30px;
+  margin-block-start: 20px;
```

Comment:
Could you explain me why we need this property `margin-block-start`? I'm just curious.